pkg/api/common.go

```diff
@@ -116,8 +119,10 @@ func ParseFlags(fs *flag.FlagSet, cfg *Config) *Config {
 	fs.StringVar(&cfg.Auth.BearerToken, "bearer-token", "", "Bearer token (JWT) used for web endpoint authentication. Disabled by default. Mutually exclusive with bearer-token-file and basic auth methods.")
 	fs.StringVar(&cfg.Auth.BearerTokenFile, "bearer-token-file", "", "Path of the file containing the bearer token (JWT) used for web endpoint authentication. Disabled by default. Mutually exclusive with bearer-token and basic auth methods.")
 
-	// PromQL configuration flags.
-	fs.StringVar(&cfg.EnableFeatures, "promql-enable-feature", "", "[EXPERIMENTAL] Enable optional PromQL features, separated by commas. These are disabled by default in Promscale's PromQL engine. "+
+	fs.StringVar(&cfg.PromscaleEnableFeatures, "enable-feature", "", "Enable beta/experimental features as a comma-separated list. Currently the following values can be passed: tracing, promql-at-modifier, promql-negative-offset")
```

Comment:
[flag package defines `Var` function](https://pkg.go.dev/flag#Var) which allows parsing string with comma-delimetered parameters into a map. With help of that function, we can reduce the `Validate` function to do just the validation and parsing could be handled the way `flag` package describes. Also doing so would remove the need to introduce another variable to hold the flags.

```suggestion
	fs.Var(&cfg.PromscaleEnableFeatures, "enable-feature", "Enable beta/experimental features as a comma-separated list. Currently the following values can be passed: tracing, promql-at-modifier, promql-negative-offset")
```
plus example 3 from https://go.dev/src/flag/example_test.go

Reply:
done